Example/Backpack/UIKit/Tokens/ColorTokensViewController.swift

```diff
@@ -0,0 +1,174 @@
+            .map(colorSection(section:))
+            .forEach(verticalStackView.addArrangedSubview)
+    }
+    
+    private func colorSection(section: Tokens.Section) -> UIView {
+        let sectionStackView = UIStackView()
+        sectionStackView.axis = .vertical
+        sectionStackView.spacing = BPKSpacingMd
+        sectionStackView.alignment = .leading
+        sectionStackView.distribution = .fill
+        sectionStackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        let sectionLabel = BPKLabel(fontStyle: .textHeading3)
+        sectionLabel.text = section.name
+        sectionStackView.addArrangedSubview(sectionLabel)
+        section.colors
+            .map(tokenView(colorToken:))
+            .forEach(sectionStackView.addArrangedSubview)
+        
+        return sectionStackView
+    }
+    
+    private func tokenView(colorToken: Tokens.Section.Color) -> UIView {
+        let container = UIView()
+        container.translatesAutoresizingMaskIntoConstraints = false
+
+        let colorView = UIView()
+        colorView.translatesAutoresizingMaskIntoConstraints = false
+        colorView.layer.cornerRadius = BPKSpacingMd
+        colorView.layer.masksToBounds = true
+        colorView.layer.borderWidth = 1
+        colorView.backgroundColor = colorToken.color
+        
+        let colorLabel = BPKLabel(fontStyle: .textLabel1)
+        colorLabel.translatesAutoresizingMaskIntoConstraints = false
+        colorLabel.text = colorToken.name
+        
+        container.addSubview(colorView)
+        container.addSubview(colorLabel)
+        
+        NSLayoutConstraint.activate([
+            colorView.leadingAnchor.constraint(equalTo: container.leadingAnchor, constant: BPKSpacingMd),
+            colorView.widthAnchor.constraint(equalToConstant: BPKSpacingLg),
+            colorView.heightAnchor.constraint(equalToConstant: BPKSpacingLg),
+            colorView.centerYAnchor.constraint(equalTo: container.centerYAnchor),
+            
+            colorLabel.topAnchor.constraint(equalTo: container.topAnchor, constant: BPKSpacingMd),
+            colorLabel.leadingAnchor.constraint(equalTo: colorView.trailingAnchor, constant: BPKSpacingMd),
+            colorLabel.bottomAnchor.constraint(equalTo: container.bottomAnchor, constant: -BPKSpacingMd),
+            colorLabel.trailingAnchor.constraint(equalTo: container.trailingAnchor, constant: -BPKSpacingMd)
+        ])
+        
+        let card = BPKCard(padded: false)
+        card.subview = container
+        return card
+    }
+}
+
+extension ColorTokensViewController {
+    struct Tokens {
+        let tokens: [Section]
+        
+        // swiftlint:disable:next nesting
+        struct Section {
+            let name: String
+            let colors: [Color]
+            
+            // swiftlint:disable:next nesting
+            struct Color {
+                let color: UIColor
+                let name: String
+            }
+        }
```

Comment:
Does this have to be nested? I don't like the swiftlint:disable calls here.

Reply:
dont like the :disable either. Nesting I do think is fine, as Section and Color are structs that have no use outside that scope, i'll move to extensions

Reply:
not fan of that rule really, as I think nesting is quite useful to scope properly, but anyways....

Reply:
We can keep it nested, should we reconsider this rule then?